### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by maximesrd**
Missing dependencies to compile Plume

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -20,13 +21,13 @@ On **Debian**:
...
 apt update
-apt install gettext postgresql postgresql-contrib libpq-dev git
+apt install gettext postgresql postgresql-contrib libpq-dev git curl gcc make openssl libssl-dev 
 ```
 
 On **Fedora**, **CentOS** or **RHEL**:
 
 ```bash
-dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel
+dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel git curl gcc make openssl libssl-dev
```
Comment (BaptisteGelez) [inline docs/INSTALL.md]: Are you sure it is called `libssl-dev`? Usually the suffix is `-devel` for RPM packages.

File: docs/INSTALL.md [hunk 2]
```diff
@@ -20,13 +21,13 @@ On **Debian**:
...
-apt install gettext postgresql postgresql-contrib libpq-dev git
+apt install gettext postgresql postgresql-contrib libpq-dev git curl gcc make openssl libssl-dev 
 ```
 
 On **Fedora**, **CentOS** or **RHEL**:
 
 ```bash
-dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel
+dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel git curl gcc make openssl libssl-dev 
 # TODO: GetText + Git install
```
Comment (BaptisteGelez) [inline docs/INSTALL.md]: Could you remove the "Git install" here since you added it?

File: docs/INSTALL.md (hunk 1 above)
Comment (maximesrd) [reply docs/INSTALL.md]: Sorry, I forgot to check.
For rpm based distrib, it's openssl-devel, and not libssl-dev

File: docs/INSTALL.md (hunk 2 above)
Comment (maximesrd) [reply docs/INSTALL.md]: Yes, I remove it, sorry

**Pull request merged**